code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1234,3 +1234,12 @@
 	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.5)
 	..()
 	. = 1
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+// END_INCLUDE
```

Comment:
wut

Reply:
I have no idea why the hell thats there, let me delete that